kernel/src/table_features/mod.rs

```diff
@@ -48,6 +51,10 @@ pub enum ReaderFeature {
     /// vacuumProtocolCheck ReaderWriter feature ensures consistent application of reader and writer
     /// protocol checks during VACUUM operations
     VacuumProtocolCheck,
+    #[serde(untagged)]
+    #[strum(default)]
+    #[strum(to_string = "{0}")]
```

Comment:
I might have missed this, but we shouldn't need to the to_string anymore

Reply:
did a quick test, and indeed we don't seem to need the extra `to_string`. strum also uses the inner value with `default` only.